CONTRIBUTING.md

```diff
@@ -1,49 +1,151 @@
+
+If the resource is not translated or if the translation is incomplete, keep the English link and translate only the description, not the title of the resource itself, because we need to make clear to the reader that it's an English document:
+```diff
+- [Using ARIA:](https://www.w3.org/TR/using-aria/) a practical guide for developers on how to add accessibility information to HTML element
++ <!-- spell ignore -->
++ [Using ARIA](https://www.w3.org/TR/using-aria/) : un guide pratique pour ajouter aux éléments HTML des informations accessibles.
+```
+
+### 4. Set non-breakable spaces
+
+`&nbsp;` is the HTML sign for "non-breakable space" and can be used in markdown as well. If the space is not set non-breakable, then a punctuation sign like ":" or ";" could be separated from the previous word and moved to the next line, which is incorrect.
+```diff
+- Using ARIA: a practical guide (...)
++ Using ARIA&nbsp;: un guide pratique (...)
+```
+
+## What tone should I use?
+
+### 1. General tone
+
+To stick to the official Guides tone, try to find the right balance to be "relax" without being too familiar:
+- Prefer "ça" over "cela"
+- Use "nous" when you mean "you & the reader", let's do this together (this is used a lot in the tutorial): "À présent, faisons ça"
+- Use "on" as the undefined pronoun, when you talk about a general rule: "En général, on écrit les variable en anglais"
+
+### 2. Inclusive writing
+
+We generally avoid the "·" character in the translation because all screen readers don't handle it very well. However, there are other ways to be inclusive in your writing. Be aware of all masculine endings and don't hesitate to rephrase the sentence in order to use a more neutral tone.
+
+```diff
+If (...), the developer would need to write the following additional code
+
+- Si (...), alors le développeur aurait dû écrire le code suivant
++ Si (...), alors nous aurions dû écrire le code suivant
 ```
 
-## Spellchecking/linting
+If you don't see how to make the sentence neutral and it's only one spot on the page, prefer the "·" character over the masculine term.
+
+### 3. Can I use Google Translate to go faster?
+
+You are free to use the method and tools you like to translate the file, what matters is the result. 
+
+Tools like [Google Translate](https://translate.google.fr/?hl=fr&sl=en&tl=fr&op=translate) can set up an approximative translation for you, then you can fix the mistakes with your own words. But keep in mind you could have _a lot_ of things to fix because it produces very literal translations. Most of the time translating yourself directly is just faster (believe me I tried).
```

Comment:
We should "recommend" DeepL instead, which offers more option and more accurate translations. Google Translate is just pure garbage, even if it's just to get an approximative translation to start with.

https://www.deepl.com

Reply:
```suggestion
Tools like [DeepL](https://www.deepl.com) can set up an approximative translation for you, then you can fix the mistakes with your own words. But keep in mind you could have _a lot_ of things to fix because it produces very literal translations. Most of the time translating yourself directly is just faster (believe me I tried).
```